framework/wazuh/core/configuration.py

```diff
@@ -32,7 +32,7 @@
 #   entry.
 #   * Last -> there can be multiple sections in the configuration but only the last one will be returned.
 #   The rest are ignored.
-conf_sections = {
+CONF_SECTIONS = {
```

Comment:
Upper case words are used for "constants". However, python dictionaries are not. If you really want to define this dictionary as constant, consider using `MappingProxyType`. If it needs to be mutable, undo this change.

Reply:
It does not need to be mutable, therefore, I will use MappingProxyType.